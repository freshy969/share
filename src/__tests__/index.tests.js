import { getType } from 'mobx-state-tree';
import Share from '../';

describe('Share', () => {
  test('networks', () => {
    const share = Share.create({});
    expect(share).toMatchSnapshot();
    expect(share.networks.map(n => getType(n).name)).toMatchSnapshot();
  });
});
