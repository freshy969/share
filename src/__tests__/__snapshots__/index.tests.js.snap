// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Share requestAllCounts 1`] = `
Array [
  Array [
    Object {
      "id": 60,
      "type": "post",
    },
  ],
]
`;
